queue_job/models/queue_job_function.py

```diff
@@ -28,6 +28,8 @@ class QueueJobFunction(models.Model):
         "related_action_enable "
         "related_action_func_name "
         "related_action_kwargs "
+        "error_handler_func_name "
```

Comment:
I think you can also introduce `error_handler_enable` in case we need to disable error handling on the fly